tta-server/app/models/Population.scala

```diff
@@ -0,0 +1,24 @@
+package models
+
+object Population {
+
+  def generateIncreasePopulationAction(playerState: PlayerState): Action = new Action{
```

Comment:
It doesn't need a playerState right now, but in the future the costToBuild will depend on the current population size.